model/postgres/postgres_resource.rb

```diff
@@ -104,6 +104,12 @@ def version
     representative_server&.version || target_version
   end
 
+  def provision_new_standby
+    exclude_host_ids, exclude_availability_zones, availability_zone = new_server_exclusion_filters
+    timeline_id = read_replica? ? parent.timeline.id : timeline.id
+    Prog::Postgres::PostgresServerNexus.assemble(resource_id: id, timeline_id:, timeline_access: "fetch", exclude_host_ids: exclude_host_ids, exclude_availability_zones: exclude_availability_zones, availability_zone: availability_zone)
```

Comment:
Hash value omission:

```suggestion
    Prog::Postgres::PostgresServerNexus.assemble(resource_id: id, timeline_id:, timeline_access: "fetch", exclude_host_ids:, exclude_availability_zones:, availability_zone:)
```